util/mqtt.go

```diff
@@ -33,17 +33,21 @@ const (
 	MqttCmdPrefix = "cmnd/"
 )
 
-func init() {
-	conf := config.GetConfig()
-	hostPort := net.JoinHostPort(conf.MqttDomain, strconv.FormatInt(int64(conf.MqttPort), 10))
-	uri := fmt.Sprintf("tcp://%s", hostPort)
-
-	opts := mqtt.NewClientOptions().AddBroker(uri)
-	opts.SetClientID(conf.ValidatorAddress)
-	opts.SetUsername(conf.MqttUser)
-	opts.SetPassword(conf.MqttPassword)
-	MQTTClient = mqtt.NewClient(opts)
+func lazyLoadMQTTClient() {
+	if MQTTClient == nil {
```

Comment:
Please consider to use "return early" here to avoid indentation

```
        if MQTTClient != nil {
                return
        }

        conf := config.GetConfig()
        hostPort := net.JoinHostPort(conf.MqttDomain, strconv.FormatInt(int64(conf.MqttPort), 10))
        uri := fmt.Sprintf("tcp://%s", hostPort)

        opts := mqtt.NewClientOptions().AddBroker(uri)
        opts.SetClientID(conf.ValidatorAddress)
        opts.SetUsername(conf.MqttUser)
        opts.SetPassword(conf.MqttPassword)
        MQTTClient = mqtt.NewClient(opts)
```